universe/node/api_objects.go

```diff
@@ -84,6 +83,26 @@ func (n *Node) apiObjectsCreateObject(c *gin.Context) {
 		}
 	}
 
+	object, ok := n.GetObjectFromAllObjects(parentID)
+	if !ok {
+		err := errors.Errorf("Node: apiObjectsCreateObject: failed to get parent object")
+		api.AbortRequest(c, http.StatusInternalServerError, "failed_to_get_parent", err, n.log)
+		return
+	}
+
+	parentObjectTypeOptions := object.GetObjectType().GetOptions()
```

Comment:
To keep it consistent: `object.GetEffectiveOptions()` ? I think (hope) that method exists.
So in theory, it could be overridden by a specific special object, the type just gives the default value.